pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1801,7 +1801,7 @@
       "BaselineVersion": "4.0.0",
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
-        "4.0.1.0": "4.3.0"
+        "4.1.0.0": "4.3.0"
```

Comment:
And TIL about the UpdatePackageIndex target.  Okay, I'm cool with things based on Eric's comments.